ImperatorToCK3/Data_Files/configurables/cultural_pillars/IRToCK3_language.txt

```diff
@@ -1984,3 +2075,234 @@ language_chong = {
 
 	color = chong
 }
+
+language_alasiyan = {
+    
+	MOD_DEPENDENT = {
```

Comment:
Missing parameters for WtWSMS: language_branch, language_family.
Missing parameters for TFE: language_group, language_family.